src/main/kotlin/com/cosmotech/api/security/AbstractSecurityConfiguration.kt

```diff
@@ -418,6 +421,11 @@ internal class CsmSecurityEndpointsRolesReader(
   }
 }
 
+@Bean
```

Comment:
Could you please remove this and test again? 
Apparently, it has no effect for the moment, so there's no need to keep it ;)